compiler/codegen/FrontEnd.hpp

```diff
@@ -200,6 +200,8 @@ class TR_FrontEnd : public TR_Uncopyable
    virtual TR_OpaqueClassBlock * getClassFromMethodBlock(TR_OpaqueMethodBlock *mb);
    virtual int32_t getNewArrayTypeFromClass(TR_OpaqueClassBlock *clazz);
 
+   virtual intptrj_t getVFTEntry(TR_OpaqueClassBlock *clazz, int32_t offset);
```

Comment:
In addition to @0xdaryl's comments it would be great if we could Doxygen document this new API so that customers of the class know what the expected behavior is.